core/src/main/java/hudson/PluginManager.java

```diff
@@ -669,9 +678,21 @@ public void dynamicLoad(File arc) throws IOException, InterruptedException, Rest
             p = strategy.createPluginWrapper(arc);
             sn = p.getShortName();
         }
-        if (getPlugin(sn)!=null)
-            throw new RestartRequiredException(Messages._PluginManager_PluginIsAlreadyInstalled_RestartRequired(sn));
-
+        PluginWrapper pw = getPlugin(sn);
+        if (pw!=null) {
```

Comment:
I agree but it seems the *rule* is being to use spaces around the operator. Anyway, this is not important. I hope to see some day a code style convention in this project 😄